grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/HttpStreamReader.java

```diff
@@ -159,7 +159,7 @@ public void onNext(HttpObject obj) {
         }
         final HttpData data = (HttpData) obj;
         try {
-            deframer.deframe(data, false);
+            deframer.deframe(data);
```

Comment:
Question: Is there any chance `data` is `endOfStream`?

Reply:
If the data's `endOfStream` is `true`, then we should handle it in `deframer`. 😉